### ISSUE TITLE
[Bug] Music videos give 403 error with local=true or local=false

### ISSUE THREAD

**Issue by DontUseGithub** [iv-org/invidious#1887]
**Describe the bug**
Any music video seems to be blocked by some magic way (youtube-dl seems to work but invidious fails)

**Steps to Reproduce**
1. Go on any invidious instance
2. Click/go on any famous Music Video like:
    (any other instance i tried gives the same error)
    https://yt.toaw.de/watch?v=lWA2pjMjpBs&local=true
    https://yt.toaw.de/watch?v=0KSOMA3QBU0&local=true
3. See error

**Logs**
The browser give the standart could not load the video inside the videoplayer
**The rest of the page still works**

**Screenshots**
 ---

**Additional context**
 - Browser (if applicable): any
 - OS (if applicable): any

**Server log**
(from yt.toaw.de)
`2021-03-13 15:04:13 UTC [info] 200 GET /watch?v=lWA2pjMjpBs&local=true 9.85ms
2021-03-13 15:04:13 UTC [info] 302 GET /latest_version?id=lWA2pjMjpBs&itag=18&local=true 3.63ms
2021-03-13 15:04:13 UTC [info] 302 GET /latest_version?id=lWA2pjMjpBs&itag=18&local=true 3.24ms
2021-03-13 15:04:13 UTC [info] 200 GET /api/v1/storyboards/lWA2pjMjpBs?height=90 4.25ms
2021-03-13 15:04:13 UTC [info] 403 GET /videoplayback?expire=1615669356&ei=DNRMYMSUJeG3gAezxbPIBA&ip=94.130.124.208&id=o-AEmS8cH2C1hJaO7NAe_HPFcYcy7rSyvbKGMVbX2Hxxb5&itag=18&source=youtube&requiressl=yes&mh=XB&mm=31%2C26&mn=sn-4g5e6nsr%2Csn-f5f7lnel&ms=au%2Conr&mv=m&mvi=4&pl=21&gcr=ua&initcwndbps=610000&vprv=1&mime=video%2Fmp4&ns=GURXAx98iordcu0lImiJoIAF&gir=yes&clen=16433849&ratebypass=yes&dur=282.911&lmt=1570267278355377&mt=1615647410&fvip=4&fexp=24001374%2C24007246&c=WEB&txp=5531432&n=f0-D7Nj7MPkKGq&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cgcr%2Cvprv%2Cmime%2Cns%2Cgir%2Cclen%2Cratebypass%2Cdur%2Clmt&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgJHBHUaV81wLyMmd8F6sTaX7X73jeyceRuT3KmDMX-hYCIQDYnkvY__nawW6ev0m9MKr9H9ikCprjm7jzPmc2IH_dVw%3D%3D&sig=AOq0QJ8wRAIgNIC5irC8_Q_Um_j0wk27AB3CuvBO-VlpkmH8wCMCVLMCICe_WPuQuHCVEHjVtGA0xkKLAkMePIZYnCvDQghq9fCC&host=r4---sn-4g5e6nsr.googlevideo.com 41.07ms
2021-03-13 15:04:13 UTC [info] 200 GET /favicon-32x32.png?v=bcb44ab 25.02ms
2021-03-13 15:04:13 UTC [info] 200 GET /api/v1/comments/lWA2pjMjpBs?format=html&hl=en-US&thin_mode=false 193.95ms
2021-03-13 15:04:14 UTC [info] 200 GET /ggpht/ytc/AAUvwniKFuwUaMmrtkj5dZbG4XcJr9RJ5CF5uVyCcQVofg=s176-c-k-c0x00ffffff-no-rj 13.62ms
`

**Additional Information**
The same video can be downloaded using the same server via youtube-dl, so chances are high that this is a invidious bug

**Comment by DontUseGithub** [comment]
So again: ONLY the googlevideo URL seems to get blocked with invidious. The rest of the page AND youtube-dl work.

**Comment by DontUseGithub** [comment]
seems to be a duplicate of:
https://github.com/iv-org/invidious/issues/1345
